packages/angular/src/migrations/update-11-1-2/update-11-1-2.ts

```diff
@@ -0,0 +1,13 @@
+import { chain } from '@angular-devkit/schematics';
```

Comment:
You can remove this migration file. It's not used.

Reply:
The chain is used in the schematic @vsavkin. It used to update the packages and formatting files

Reply:
I mean we don't need the whole file.

Packages are updated automatically based on the versions specified in migrations.json

Reply:
Thats good to know. I will update the PR